modules/core/src/weaver/internals/ExpectyListener.scala

```diff
@@ -1,11 +1,13 @@
 package weaver
+package internals
 
 import cats.data.{ NonEmptyList, ValidatedNel }
 import cats.syntax.all._
 
 import com.eed3si9n.expecty._
 
-class ExpectyListener extends RecorderListener[Boolean, Expectations] {
+private[weaver] class ExpectyListener
```

Comment:
I remember you asked me on the scala 3 PR if making it package private will work outside of weaver package (and it does, but we're not sure why)

Reply:
Are you not confusing the `ExpectyListener` class with the `Expecty` class ?

Reply:
![image](https://user-images.githubusercontent.com/1052965/104162784-26e79d80-53ed-11eb-91e9-4786326457b4.png)